web-client/src/presenter/computeds/AdvancedSearch/practitionerSearchHelper.ts

```diff
@@ -78,7 +79,8 @@ export const practitionerSearchHelper = (
     result = {
       ...result,
       formattedSearchResults: paginatedResults,
-      numberOfResults: Number(formatPositiveNumber(searchResults.total)),
+      numOfResultsDisplay: formatPositiveNumber(searchResults.total),
+      numberOfResults: searchResults.total,
```

Comment:
I'm not a fan of returning the same value twice, once as a number, and the other as a string. Can you please revert this so we format the number in the view as that's a better solution than this IMO.

Reply:
Or look at solving this differently? I might have sometime later this afternoon to help if needed.